tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Do we need that new line?

Reply:
I think so because the two paragraphs talk about different things.